hera_mc/cm_utils.py

```diff
@@ -16,6 +16,17 @@
 PAST_DATE = '2000-01-01'
 
 
+class utility_method:
```

Comment:
While the whole argparse module is tuned to arg parsing, its Namespace class is a generic helper that's *exactly* what you're creating. It can be initialized pretty much the way you use yours:

```
>>> from argparse import Namespace
>>> ns = Namespace(foo='bar', other=123)
>>> ns
  < Namespace(foo='bar', other=123)
>>> ns.foo
  < 'bar'
```

The advantage of the existing Namespace object is that it reuses existing infrastructure instead of reimplementing it, and uses standard terminology, and has built-in support for other standard operations like str() and `==`.